### ISSUE TITLE
[Windows 7] --screenshot-template not working properly

### ISSUE THREAD

**Issue by KennyMan666** [mpv-player/mpv#2201]
I started using mpv today and managed to solve the immediate issues I had with it, coming over from mplayer (mostly input stuff). The only one that remains right now is saving screenshots. I managed to get --screenshot-directory working, but that doesn't seem to do relative locations, and for some reason --screenshot-template refuses to work properly. The primary issue I had was that I don't want all screenshots saved in one folder - I want a screenshot to be saved in the same folder as the video file, something I'm pretty sure mplayer did by default (I don't remember ever configuring it to do that, at least). Currently -template only works at all if I start it with a \, but that makes all screenshots get saved to the root of the disk. If I put a %x after the \, I get error writing screenshot, and if I put it before, it ignores -template entirely.

What's with this function and how do I get it to behave?

**Comment by lachs0r** [comment]
Did you try just setting `screenshot-directory` to an empty string (`--screenshot-directory=''`)?

**Comment by wm4** [comment]
Use forward slashes.

**Comment by wm4** [comment]
Actually, I misunderstood your problem.

mpv's behavior is the same as mplayer, if it's started as mpv.com from the command line. Otherwise, this applies: http://mpv.io/manual/master/#pseudo-gui-mode

**Comment by KennyMan666** [comment]
Yeah, I read the manual, it came with the Windows build I got and [pseudo-gui] was one of those things I had to add to make the directory line work. lachs0r's suggestion about the directory worked, the screenshots get saved to the same folder as the movie file, but they're still named mpv-shot%n. So the template function is still not working, for whatever reason.

My template line currently reads --screenshot-template=%F-%wHh%wMm%wSs, if that helps.
